doc/man3/PKCS12_newpass.pod

```diff
@@ -17,6 +17,16 @@ PKCS12_newpass() changes the password of a PKCS12 structure.
 B<p12> is a pointer to a PKCS12 structure. B<oldpass> is the existing password
 and B<newpass> is the new password.
 
+=head1 NOTES
+
+Both B<oldpass> and B<newpass> are NUL-terminated strings in the UTF-8 encoding.
+If either is not a valid UTF-8 byte sequence, then that one is interpreted
```

Comment:
How about "Each of B<oldpass> and B<newpass> is independently interpreted as a string in the UTF-8 encoding. If it is not valid UTF-8, it is assumed to be ISO8859-1 instead".

Mostly I've just add 'independently' to your suggestion...